doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,188 @@
+
+**Objective**<br>
+Specify Kubernetes infrastructure and CNF Conformance aspects using requirements in CNTT RA2 and RM.
+
+Upstream projects define features/capabilities, test scenarios, and test cases which may be used for infrastructure and/or CNF verification and validation. Where feasible existing test frameworks, test methods and tests may also be used be used for conformance testing along with new tests developed where there are gaps.  
+
+Many existing tests currently being considered for RC-2 are functional tests from upstream projects. Reuse of existing test suites is encouraged however OVP 2.0 is not aiming to replace programs such as Certified Kubernetes which is a "CNCF software conformance program that ensures a vendor’s version of Kubernetes supports the required APIs, as do open source community version". 
+
+The focus of conformance is to test the actual behavior of the system with respect to a capability specified in the RM or RA-2. 
+
+
+**Test Methodology**
+- Verification test to make sure that the Kubernetes services have been deployed and configured correctly
+- Manifest Verifications (Termed Compliance by CVC) will ensure the infrastructure is compliant, and delivered for testing, with hardware and software profile specifications defined by the RM and RA2
+- Candidate CNF Validation will ensure interoperablity of CNF behaviour on the Kubernetes infrastructure.  Testing ensures CNFs can be spun up, modified, or removed, on the target infrastructure (aka Interoperability).  
+
+<a name="1.1.1"></a>
+### 1.1.1 Terminology
+
+Terminology in this document will follow [CNTT Terminology](../../../tech/glossary.md). Relevant testing terminology includes:
+
+<a name="1.2"></a>
+## 1.2 Scope
+
+This document covers aspects of CNTT conformance for both Kubernetes based cloud infrastructure and CNFs. The document will cover the following topics:
+
+- Identify in details the requirements of test-cases (mapped from RA2 and RM)
+- Test criteria that shows a certain capability or feature of the system-under-test exists and behaves as expected
+- An E2E framework for conformance of Kubernetes infrastructures and CNFs, including specification for conformance test infrastructure (lab environment and tools)
+- Analysis to identify where the gaps are in the industry for implementing conformance test objectives (tooling, methods, process, etc).
+
+**Not in Scope**
+- Functional testing / validation of the application provided by the CNF is outside the scope of this work.
+- Testing to confirm specific versions of APIs or software components is out of scope of conformance since this is an unambiguous part of a product specification.
```

Comment:
this is confusing

Reply:
which part? can you make a suggestion?

Reply:
Clarification requested on:
> Testing to confirm specific versions of APIs or software components is out of scope of conformance since this is an unambiguous part of a product specification.

Reply:
The goal for the RC2 conformance is verifying the actual behaviour and capabilities of a running platform.

Versions are an indication that a behaviour or capability may be there, but it needs to be actually tested.